appyx-components/demos/android/src/main/kotlin/com/bumble/appyx/components/demos/cards/DatingCards.kt

```diff
@@ -41,57 +35,43 @@ fun DatingCards(modifier: Modifier = Modifier) {
                 },
                 savedStateMap = null
             ),
-            motionController = { CardsMotionController(it)  },
+            motionController = { CardsMotionController(it) },
             gestureFactory = { CardsMotionController.Gestures(it) },
             animateSettle = true
         )
     }
 
     InteractionModelSetup(cards)
 
-    Children(
+    DraggableChildren(
         modifier = modifier
             .fillMaxSize()
             .background(appyx_dark)
             .padding(16.dp),
+        screenWidthPx = (LocalConfiguration.current.screenWidthDp * LocalDensity.current.density).roundToInt(),
+        screenHeightPx = (LocalConfiguration.current.screenHeightDp * LocalDensity.current.density).roundToInt(),
```

Comment:
I guess this could be moved one level deeper, so that client code doesn't need to specify it.